flowredux/src/commonMain/kotlin/com/freeletics/flowredux/dsl/FlowReduxStateMachine.kt

```diff
@@ -50,6 +53,10 @@ public abstract class FlowReduxStateMachine<S : Any, A : Any>(
             return outputState
         }
 
+    public fun stateAndInitialState(): StateWithInitialState<S> {
+        return StateWithInitialState(state, currentState ?: initialState)
+    }
```

Comment:
We could also have our one `@OptIn` annotation to mark this as internal so that it's only used from the compose artifact.